superset-frontend/src/SqlLab/components/SqlEditor/SqlEditor.test.jsx

```diff
@@ -263,8 +262,8 @@ describe('SqlEditor', () => {
   it('render a Limit Dropdown', async () => {
     const defaultQueryLimit = 101;
     const updatedProps = { ...mockedProps, defaultQueryLimit };
-    const wrapper = buildWrapper(updatedProps);
-    await waitForComponentToPaint(wrapper);
-    expect(wrapper.find(AntdDropdown)).toExist();
+    const { findByText } = setup(updatedProps, store);
+    fireEvent.click(await findByText('LIMIT:'));
```

Comment:
> Most projects have a few use cases for fireEvent, but the majority of the time you should probably use [@testing-library/user-event](https://testing-library.com/docs/user-event/intro).

Should we use `user-event` here?

Reply:
I think we should!  I will switch that over and push up the changes.